src/ontology/doid-edit.owl

```diff
@@ -40307,6 +40318,88 @@ AnnotationAssertion(rdfs:label obo:DOID_0060911 "karyomegalic interstitial nephr
+AnnotationAssertion(rdfs:label obo:DOID_0060912 "craniosynostosis 7"@en)
+SubClassOf(obo:DOID_0060912 obo:DOID_2340)
+SubClassOf(obo:DOID_0060912 ObjectSomeValuesFrom(obo:IDO_0000664 obo:GENO_0000930))
+
+# Class: obo:DOID_0060913 (Usher syndrome type IIC)
+
+AnnotationAssertion(obo:IAO_0000115 obo:DOID_0060913 "A Usher syndrome characterized by  by a homozygous or compound heterozygous mutation in the ADGRV1 gene (602851) on chromosome 5q14. It is also caused by biallelic digenic mutation in the ADGRV1 and PDZD7 (612971) genes.")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060913 "OMIM:605472")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060913 "USH2C")
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:DOID_0060913 "disease_ontology")
+AnnotationAssertion(oboInOwl:id obo:DOID_0060913 "DOID:0060913")
+AnnotationAssertion(rdfs:label obo:DOID_0060913 "Usher syndrome type IIC"@en)
+SubClassOf(obo:DOID_0060913 obo:DOID_0050439)
+SubClassOf(obo:DOID_0060913 ObjectSomeValuesFrom(obo:IDO_0000664 obo:GENO_0000930))
+
+# Class: obo:DOID_0060914 (Proteasome-associated autoinflammatory syndrome 1)
+
+AnnotationAssertion(obo:IAO_0000115 obo:DOID_0060914 "This autosomal recessive systemic autoinflammatory disorder is characterized by early childhood onset of annular erythematous plaques on the face and extremities with subsequent development of partial lipodystrophy and laboratory evidence of immune dysregulation. More variable features include recurrent fever, severe joint contractures, muscle weakness and atrophy, hepatosplenomegaly, basal ganglia calcifications, and microcytic anemia.")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060914 "OMIM:256040")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060914 "PRAAS1")
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:DOID_0060914 "disease_ontology")
+AnnotationAssertion(oboInOwl:id obo:DOID_0060914 "DOID:0060914")
+AnnotationAssertion(rdfs:label obo:DOID_0060914 "Proteasome-associated autoinflammatory syndrome 1"@en)
+SubClassOf(obo:DOID_0060914 obo:DOID_0050737)
+SubClassOf(obo:DOID_0060914 obo:DOID_0060919)
+SubClassOf(obo:DOID_0060914 ObjectSomeValuesFrom(obo:IDO_0000664 obo:GENO_0000930))
+
+# Class: obo:DOID_0060916 (Proteasome-associated autoinflammatory syndrome 3)
+
+AnnotationAssertion(obo:IAO_0000115 obo:DOID_0060916 "Proteasome-associated autoinflammatory syndrome-3 is an autosomal recessive syndrome with onset in early infancy. Affected individuals present with nodular dermatitis, recurrent fever, myositis, panniculitis-induced lipodystrophy, lymphadenopathy, and dysregulation of the immune response, particularly associated with abnormal type I interferon-induced gene expression patterns. Additional features are highly variable, but may include joint contractures, hepatosplenomegaly, anemia, thrombocytopenia, recurrent infections, autoantibodies, and hypergammaglobulinemia. Some patients may have intracranial calcifications.
```

Comment:
Fix definition by reducing length and fitting to pattern: "A(n) {parent term} characterized by {symptoms, phenotypes, onset} that has_material_basis_in {genetic cause}."

Reply:
Great!